From 6c06ac0571fb6bf0734b173cc3a75badc7554601 Mon Sep 17 00:00:00 2001
From: Shwetha K Acharya <sacharya@redhat.com>
Date: Tue, 14 Jan 2020 10:51:06 +0530
Subject: [PATCH 350/350] tools/glusterfind: Remove an extra argument

Backport of:
>    Upstream Patch: https://review.gluster.org/#/c/glusterfs/+/24011/
>    fixes: bz#1790748
>    Change-Id: I1cb12c975142794139456d0f8e99fbdbb03c53a1
>    Signed-off-by: Shwetha K Acharya <sacharya@redhat.com>

Change-Id: I1cb12c975142794139456d0f8e99fbdbb03c53a1
BUG: 1789447
Signed-off-by: Sunny Kumar <sunkumar@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/189363
Tested-by: RHGS Build Bot <nigelb@redhat.com>
Reviewed-by: Sunil Kumar Heggodu Gopala Acharya <sheggodu@redhat.com>
---
 tools/glusterfind/src/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/glusterfind/src/main.py b/tools/glusterfind/src/main.py
index dfc9d07..5ca1fec 100644
--- a/tools/glusterfind/src/main.py
+++ b/tools/glusterfind/src/main.py
@@ -515,7 +515,7 @@ def write_output(outfile, outfilemerger, field_separator):
                     continue
 
                 if row_2_rep and row_2_rep != "":
-                    gfind_write_row(f, row[0], field_separator, p_rep, field_separator, row_2_rep)
+                    gfind_write_row(f, row[0], field_separator, p_rep, row_2_rep)
 
                 else:
                     gfind_write(f, row[0], field_separator, p_rep)
-- 
1.8.3.1

